src/cl/lodestar/lodestar_launcher.star

```diff
@@ -327,7 +327,7 @@ def get_beacon_config(
             + constants.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER
             + "/genesis.ssz"
         )
-        if network == "kurtosis":  # kurtosis
+        if network == "kurtosis":  # Kurtosis
```

Comment:
do we need the comment?

Reply:
no, not really, it was really just for me to be able to follow what I was doing. Can remove it.

Reply:
Lets clean it up!

Reply:
done